ocs/ocp.py

```diff
@@ -88,6 +92,8 @@ def get(self, resource_name='', out_yaml_format=True, selector=None):
             dict: Dictionary represents a returned yaml file
         """
         command = f"get {self.kind} {resource_name}"
+        if all_namespaces and not self.namespace:
+            command += "-A"
```

Comment:
Why all namespaces ? If openshift creates OR deletes its house keeping resources even that would be counted right ? And if user wants to do garbage collection(assuming if user gets leftovers exception and wants to clean up the garbage) I feel it could lead to disaster. Am I missing something ?

Reply:
- > Why all namespaces ? If openshift creates OR deletes its house keeping resources even that would be counted right ?
You are right about the first part - I would not expect the OCP will add new entities just like that without any request
 And if user wants to do garbage collection(assuming if user gets leftovers exception and wants to clean up the garbage) I feel it could lead to disaster. Am I missing something ?
So the user should fix the teardown (:

Reply:
Yep, that's why I said house keeping, which may not require any external request although I am not sure about any specific case.